packages/nextjs/src/app-router/client/useAwaitableReplace.ts

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import { usePathname, useRouter } from 'next/navigation';
+import { useCallback, useEffect, useTransition } from 'react';
+
+import type { NextClerkProviderProps } from '../../types';
+
+declare global {
+  interface Window {
+    __clerk_internal_navPromisesBuffer: Array<() => void> | undefined;
+    __clerk_internal_navFun: NonNullable<NextClerkProviderProps['routerPush']>;
```

Comment:
Seems like we are redeclaring these types twice as they are identical to the original ones.
Should we update to this ?
```suggestion
    __clerk_internal_navFun: NonNullable<NextClerkProviderProps['routerPush'] | NextClerkProviderProps['routerReplace']>;
```

Reply:
missed this, thanks